meta-lmp-bsp/recipes-security/optee/optee-os-tadevkit_3.17.0.bbappend

```diff
@@ -1,3 +1,3 @@
 # Do not assume one by default as we also support different providers
-require ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os-fio', 'optee-os-fio-bsp.inc', '', d)}
-require ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os-fio-mfgtool', 'optee-os-fio-bsp-mfgtool.inc', '', d)}
+include ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os-fio', 'optee-os-fio-bsp.inc', '', d)}
+include ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os-fio-mfgtool', 'optee-os-fio-bsp-mfgtool.inc', '', d)}
```

Comment:
I think here having require is ok, as those files are all related to configuration and under the bsp layer.